src/Assets/Schemas/ExtensionInformation.schema.json

```diff
@@ -0,0 +1,94 @@
+{
+  "$id": "",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "description": "Required schema to allow users to discover non-installed extensions within Dev Home.",
+  "type": "object",
+  "properties": {
+    "Products": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "ProductId": { "type": "string" },
+          "Properties": {
+            "type": "object",
+            "properties": {
+              "PackageFamilyName": { "type": "string" },
+              "LocalizedProperties": {
+                "type": "object",
+                "properties": {
+                  "DisplayNameKey": { "type": "string" },
+                  "PublisherDisplayNameKey": { "type": "string" }
+                },
+                "required": [ "DisplayNameKey", "PublisherDisplayNameKey" ]
+              },
+              "DevHomeExtensions": {
+                "type": "array",
```

Comment:
Yes, Dev Home itself would be considered one. We have one package/Product but we have the WSL extension, HyperV extension the Git extension as well. Looking at this I figured it would be best to account for these scenarios as well.